homeassistant/components/intent/__init__.py

```diff
@@ -358,6 +372,7 @@ def __init__(self) -> None:
             },
             description="Sets the position of a device or entity",
             platforms={COVER_DOMAIN, VALVE_DOMAIN},
```

Comment:
For a future PR, we might want to make `platforms` a dictionary mapping platform to device class.